.github/workflows/labeler-missing-area-label.yml

```diff
@@ -0,0 +1,63 @@
+name: "Spot Issues Missing Area Labels"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 13 * * *' # Runs daily at 13:00 UTC
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  check-area-labels:
+    name: "Check Issues Missing Area Labels"
+    runs-on: ubuntu-latest
+
+    if: github.repository == 'dotnet/roslyn' # Skip forks
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Tag Issues Missing Area Labels
+        uses: actions/github-script@v6
```

Comment:
Looks like the latest is [v7](https://github.com/actions/github-script) can we use it instead?